### ISSUE TITLE
MyAlerts v2.0.x support?

### ISSUE THREAD

**Issue by Eldenroot** [MyAlerts/Custom-Alerts-for-MyAlerts#10]
MyAlerts v2.0.x support?

**Comment by Shade-** [comment]
I do not plan to update CAFM anytime soon as I'm currently working on [MyChat](http://projectxmybb.altervista.org).

**Comment by keithpickering** [comment]
It seems this plugin is 100% incompatible with MyBB 1.8 and MyAlerts 2. Apparently the dev will be away until July, so I'm going to try rebuilding it.

**Comment by Shade-** [comment]
Yes indeed. The code might look awful though. I planned a rewrite but given my assignments it would be awesome if you could rewrite it from scratch, maybe using an OOP approach.
